### DISCUSSION TITLE
Why are the profiling flamegraphs displayed in the tags section instead of with the rest of the spans?

### DISCUSSION THREAD

**Discussion by zachauten** [grafana/pyroscope#1705] [Q&A]
This might be a dumb question, because i'm totally new to Pyroscope, but I was wondering why the flame graphs from the profiler in the [open telemetry examples](https://github.com/pyroscope-io/pyroscope/tree/main/examples/tracing/jaeger/) are displayed separately from the other spans. Is it related to open telemetry [not having finalized a spec for profiling](https://github.com/open-telemetry/oteps/issues/139) yet?
For reference, what i'm trying to do is send profiling data from the Pyroscope agent to an open telemetry collector via OTLP, which should in turn send it to the observability backend (in my case Datadog). I'm not sure if this is possible with Pyroscope at the moment, and i"m pretty sure I don't have a complete/correct mental model for how Pyroscope fits into the open telemetry ecosystem yet. Any clarification would be appreciated!

**Comment by cyriltovena** [comment]
👋  Zach !

Very late answer after a year. But you were right, OTEL is not mature enough and so the support is limited for Profiling. Pyroscope will be a backend like datadog to receive the data and visualize it later.

You can follow work on the OTEL profiling side here https://github.com/open-telemetry/oteps/pull/237.

More than a year after you question this is still work in progress.
